index.js

```diff
@@ -19,30 +19,84 @@ function start(app) {
 
   pm.start(run);
 
-  function run(err) {
+  function run(err, logAt) {
     if (err) {
       console.error('%s', err);
-      process.exit(1);
+      return done(err);
     }
 
     debug('deploy to %j app %j', pm.url, app);
-    deploy(
-      {
-        baseURL: pm.url,
-        serviceName: name,
-        branchOrPack: app,
-        clusterSize: 1,
-      },
-      ran);
+    deploy({
+      baseURL: pm.url,
+      serviceName: name,
+      branchOrPack: app,
+      clusterSize: 1,
+    }, function(err, service) {
+      ran(err, service, logAt);
+    });
   }
 
-  function ran(err, service) {
+  function ran(err, service, logAt) {
     assert.ifError(err);
     var msg = fs.readFileSync(require.resolve('./started.txt'), 'utf-8')
-      .replace(/%APP%/g, app)
-      .replace(/%SVC%/g, service.name || service.id)
-      .trim();
+    .replace(/%APP%/g, app)
+    .replace(/%SVC%/g, service.name || service.id)
+    .trim();
     console.log(msg);
-    process.exit();
+    console.log('');
+    console.log('--- tail of %s ---', pm.logFile);
+    tail(pm.logFile, logAt, function() {
+      return done();
+    });
   }
 }
+
+function tail(file, logAt, callback) {
+  var buf = new Buffer(4096);
+  fs.open(file, 'r', function(err, fd) {
+    // If you removed your log file... you don't get logs.
+    if (err)
+      return callback();
+
+    var quiet = false;
+
+    read(timeout);
+
+    function timeout(err, data) {
+      // If we errored, we're done.
+      if (err)
+        return callback();
+
+      // If we read data, keep reading.
+      if (data) {
+        quiet = false;
+        return process.stdout.write(data, function() {
+          read(timeout);
```

Comment:
Gotcha. Seems reasonable then, as long as people aren't afraid to kill their app by hitting ctrl+c.